miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt

```diff
@@ -31,14 +32,20 @@ internal fun createRetrofitClient(
     @Suppress("SpreadOperator")
     val httpClient = OkHttpClient.Builder()
         .addHeaderInterceptor(*headers.asArray())
+        .addInterceptor(provideHeaderInterceptor())
```

Comment:
I think you should be able to use something like `.addHeaderInterceptor("Accept-Encoding" to "identity")` instead of manually defining your own interceptor. `.addHeaderInterceptor` is an extension function from the SDK Utils library (https://github.com/rakutentech/android-sdkutils#okhttp-header-interceptor).

Reply:
`.addHeaderInterceptor("Accept-Encoding" to "identity")` has no affect 🤔 . There is different that our sdk utils lib uses `addNetworkInterceptor`, not `addInterceptor`.

Reply:
Ah, okay yeah you're right. A network interceptor is different than a normal interceptor.